types/ltijs/lib/Provider/Provider.d.ts

```diff
@@ -42,6 +42,10 @@ export interface OnConnectCallback {
     (connection: IdToken, request: Request, response: Response, next: NextFunction): Response | void;
 }
 
+export interface OnRequestCallback {
```

Comment:
I'd use a better fitting name here. In the original library, `unregisteredPlatformCallback` is used. What do you think?

Reply:
Perfect I will change it